lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -75,9 +73,9 @@ extension LocalizedBody on Event {
   bool get isContainsLink => firstValidUrl != null;
 
   void shareFile(BuildContext context) async {
-    final matrixFile = await getFile(context);
+    final matrixFile = await downloadAndDecryptAttachment();
```

Comment:
this case also freeze mobile app. please keep the current one, or wait until we complete grooming for DownloadManager